build.gradle.kts

```diff
@@ -18,7 +18,7 @@ dependencies {
     // https://chat.fhir.org/#narrow/stream/179166-implementers/topic/New.20validator.20JAR.20location
     // the ig-publisher uses this one too
     // https://github.com/HL7/fhir-ig-publisher/blob/master/pom.xml#L68
-    implementation("ca.uhn.hapi.fhir", "org.hl7.fhir.validation", "5.5.10")
+    implementation("ca.uhn.hapi.fhir", "org.hl7.fhir.validation", "5.6.21")
```

Comment:
I am confused why it thinks it is being changed from `5.5.10` to `5.6.21`, when `master` already has `5.6.21`?  I assume confusion from merging from master into this branch instead of rebasing?  Rebasing is not as easy as merging back through github, but wierd things like this are a pain for reviewers and rebasing keeps diffs very clean.  No actions needed (except try to rebase in the future!)